android/src/test/java/com/amplitude/android/AmplitudeOtherTest.kt

```diff
@@ -0,0 +1,51 @@
+package com.amplitude.android
```

Comment:
Can you please give this a more descriptive name than "AmplitudeOtherTest.kt"? Something like 
 "AmplitudeAndroidTests.kt" or "AmplitudeRobolectricTests.kt" so we can easily add future tests to the correct file.

Reply:
Ha, that's a good suggestion. I'm also a little bit struggling with the test name since we already have AmplitudeTest.kt.

Reply:
Changed it to `AmplitudeRobolectricTests`.